### PR TITLE
Progressive web app

### PR DESCRIPTION
**Pull request by anze3db**
This change makes the app installable on mobile:
screenshot_20160331-225054
And it also moves package.json to the root folder, so that you don't have to CD into public everytime you want to run the app. I hope you don't mind.

### CODE REVIEW

File: public/manifest.json [hunk 1]
```diff
@@ -0,0 +1,24 @@
+{
+  "short_name": "Reci Food",
+  "name": "Reciprocity's awesome food app",
```
Comment (edofic) [inline public/manifest.json]: Can you make this non-Reci specific? I'm trying real hard to keep the app unbranded. Thanks

Comment (anze3db) [inline public/manifest.json]: Of course.

**Comment by edofic** [comment]
Thanks! Will deploy in a few minutes

**Comment by edofic** [comment]
@Smotko my instance now live. Enjoy :smile:

**Comment by anze3db** [comment]
yay! 👍

**Pull request merged**